src/node_version.h

```diff
@@ -93,7 +93,10 @@
 
 // The NAPI_VERSION provided by this version of the runtime. This is the version
 // which the Node binary being built supports.
-#define NAPI_VERSION 9
+// Distinguished from NAPI_VERSION intentionally to avoid re-defining
```

Comment:
```suggestion
// The version of Node-API built into this instance of Node.js.
// Distinguished from NAPI_VERSION intentionally to avoid re-defining
```